packages/tasks/src/model-libraries.ts

```diff
@@ -980,6 +980,15 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		countDownloads: `path_extension:"pt" OR path_extension:"safetensors"`,
 		snippets: snippets.ultralytics,
 	},
+	zonos: {
+		prettyLabel: "Zonos",
+		repoName: "Zonos",
+		repoUrl: "https://github.com/Zyphra/Zonos",
+		docsUrl: "https://github.com/Zyphra/Zonos",
+		snippets: snippets.zonos,
+		filter: false,
+		countDownloads: `path:"model.safetensors"`,
```

Comment:
seems like downloads are already counted on the `config.json` file (e.g. https://huggingface.co/Zyphra/Zonos-v0.1-hybrid/blob/main/config.json) so better not to change that now (for consistency + it already works)

Reply:
```suggestion
```